server/middleware/setUpSentry.ts

```diff
@@ -28,19 +22,13 @@ function setUpSentryRequestHandler(app: express.Express): void {
       },
       dsn: config.sentry.dsn,
       environment: config.environment,
-      integrations: [
-        // enable HTTP calls tracing
-        new Sentry.Integrations.Http({ tracing: true }),
-        // enable Express.js middleware tracing
-        new Sentry.Integrations.Express({ app }),
-      ],
       release: gitRef,
       tracesSampler: samplingContext => {
-        const transactionName = samplingContext?.transactionContext?.name
```

Comment:
As per https://docs.sentry.io/platforms/javascript/migration/v8-to-v9/:

`samplingContext.transactionContext.name` has been replaced with `samplingContext.name`